src/tools/recipe2plan-cli.ts

```diff
@@ -72,7 +73,10 @@ void Flags.withDefaultReferenceMode(async () => {
     fs.mkdirSync(opts.outdir, {recursive: true});
 
     const manifest = await Runtime.parseFile(opts._[0]);
-    const plans = await recipe2plan(manifest, outFormat, opts.recipe);
+    // TODO(b/159144612): Make policies manifest parameter mandatory.
```

Comment:
I don't think that's required. For Arcs clients in general, policies aren't a mandatory thing. You should still be able to generate a plan without first writing a policy.

Reply:
removed (and updated parameters order, as policiesManifest is now officially optional)